src/app/tokens/hooks/useERC20Tokens.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from "ethers";
 import { useCallback, useEffect, useMemo, useState } from "react";
 
-import { ERC20ABI } from "../constants";
+import { ERC20ABI } from "@/app/constants";
```

Comment:
we should to create a sub folder to manage the project abis

Reply:
nicely done